packages/web_benchmarks/lib/src/recorder.dart

```diff
@@ -361,19 +359,20 @@ abstract class WidgetRecorder extends Recorder implements FrameRecorder {
   Widget createWidget();
 
   final List<VoidCallback> _didStopCallbacks = <VoidCallback>[];
+
   @override
   void registerDidStop(VoidCallback fn) {
     _didStopCallbacks.add(fn);
   }
 
   @override
-  Profile profile;
-  Completer<void> _runCompleter;
+  late Profile profile;
+  Completer<void>? _runCompleter;
```

Comment:
This is force-unwrapped a lot; please add a comment explaining which methods it's guaranteed to be non-null in and why.